projects/wt/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN apt-get update && apt-get install -y libboost-all-dev cmake
+RUN git clone --depth=1 https://github.com/0x34d/wt
```

Comment:
This will be changed after https://github.com/emweb/wt/pull/203 merge

Reply:
The OSS-Fuzz panel approved this as well. Do you happen to have any updates from the project owners?

Reply:
Well, till now it's a dead END.

Reply:
Feel free to close this PR if you decide to move on : )
Otherwise, I am happy to leave it open as a draft in case the project owners agree later.